ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -179,7 +179,48 @@ scp "${SSHOPTS[@]}" "root@${IP}:/tmp/ds-vars.conf" "${SHARED_DIR}/"
 # Add required configurations ci-chat-bot need
 ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF |& sed -e 's/.*auths.*/*** PULL_SECRET ***/g'
 echo "https://\$(oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > /tmp/console.url
+
+echo "Collecting cluster data for analysis..."
+set +o errexit
+set +o pipefail
+if [ ! -f /tmp/jq ]; then
+  curl -L https://stedolan.github.io/jq/download/linux64/jq -o /tmp/jq && chmod +x /tmp/jq
```

Comment:
I would prefer we don't introduce any new curls to GitHub.  We often get throttled and blocked, and when it happens all these things stop working and jobs fail.  `oc` can already do this directly anyway, see below